packages/shared/src/components/Logo.tsx

```diff
@@ -98,11 +106,19 @@ export default function Logo({
         href={webappUrl}
         onClick={onLogoClick}
       >
-        <LogoSvgElem
-          className={logoClassName}
-          src={featureTheme?.logo}
-          fallback={LogoIcon}
-        />
+        <div className="relative">
+          <LogoSvgElem
+            className={logoClassName}
+            src={featureTheme?.logo}
+            fallback={LogoIcon}
+          />
+          {isPlus && (
+            <DevPlusIcon
+              className="text-action-plus-default absolute right-0 top-0 -translate-y-2/3 translate-x-2/3"
+              size={IconSize.XXSmall}
+            />
+          )}
+        </div>
```

Comment:
SVG in figma was two separate icons, so couldn't do the same method as with LogoText